doc/sphinxman/source/dlpnomp2.rst

```diff
@@ -0,0 +1,135 @@
+
+.. codeauthor:: Zach Glick 
+.. sectionauthor:: Zach Glick
+
+*Module:* :ref:`Keywords <apdx:dlpnomp2>`, :ref:`PSI Variables <apdx:dlpnomp2_psivar>`, :source:`DLPNOMP2 <psi4/src/psi4/dlpnomp2>`
+
+Introduction
+------------
+
+The steep polynomial scaling (in both time and memory) of post-HF dynamic
+correlation methods prohibits calculations on large systems, even for 
+efficient codes like |Psi4four|'s DFMP2. This poor scaling is in part due to
+the use of canonical HF orbitals, which are entirely delocalized across the 
+molecule, to compute the correlation energy. Canonical orbitals are usually
+used for their mathematical convencience. Another possible choice is localized
+orbitals. any two orbitals localized to separate regions of a molecule
+can be treated as non-interacting to a good approximation. Thus, when working
+with localized orbitals, the number of interacting orbital pairs (and triples, 
+quadruples, etc.) scales linearly with system size. If carefully implemented, 
+programs that exploit this sparsity can be made to scale linearly (or else with 
+lower order than their canonical counterparts) at the cost of of modest, 
+controllable errors. This is the defining insight of DLPNO-MP2 and all "local
+correlation" methods.
+
+The DLPNO-MP2 code is a linear-scaling alternative to the :ref:`DF-MP2 <sec:dfmp2>`
+code, and is intended for use with large systems for which DF-MP2 is intractable. 
+When running DLPNO-MP2 with default settings, approximately 99.9% of the DF-MP2 
+correlation energy is recovered. The general outline of the method is as follows:
+
+    (1) Localize the canonical HF orbitals: 
+        (a) Perform Foster-Boys localization on the active occupied MOs
+        (b) Construct Projected AOs from the virtual MOs
+    (3) Calculate three-index integrals in (sparse) LMO/PAO basis
+    (3) Perform local density fitting
+    (4) Transform local virtual space from PAOs to truncated PNOs 
+    (5) Solve the iterative local MP2 equations in the LMO/PNO basis
+
+An example input file is::
+
+   molecule h2o {
+   0 1
+   O
+   H 1 1.0
+   H 1 1.0 2 104.5
+   symmetry c1
+   }
+   
+   set basis cc-pvdz
+   set scf_type df
+   set freeze_core True
+   
+   energy('dlpno-mp2')
+
+The only difference between this input and a DF-MP2 input is the ``energy('dlpno-mp2')``
+call to :py:func:`~psi4.energy`. Note that the water molecule in this example is far too 
+small for DLPNO-MP2 to be of any benefit relative to DF-MP2.
+
+
+The theory, thresholds, and best practices associated with DLPNO-MP2 are presented below. 
+
+.. index::
+   pair: DLPNO-MP2; theory
+
+Theory
+------
+
+See :ref:`DF-MP2 <sec:dfmp2>` for background on the theory of (non-local) density-fitted MP2.
+
+TODO
+
+Recommendations
+---------------
+
+Some practical notes on running the code:
+
+* DLPNO-MP2 should be used for large calculations that cannot reasonable be performed
+  with DF-MP2 (i.e. > X basis functions).
+
```

Comment:
Could you please add something along the lines of below (no need to take my exact phrase):

```
* The PNO accuracy can be controlled with the |dlpno__PNO_CONVERGENCE| keyword according to recommendation by Liakos et al. [Liakos:2015:1525|_. For non-covalent interactions ``TIGHT`` is highly recommended. 
```
then http://dx.doi.org/10.1021/ct501129s would need to be added to `bibliography.rst`

Reply:
Sure, thanks for the paper